src/pint/models/binary_ell1.py

```diff
@@ -265,6 +243,8 @@ def __init__(self):
                 description="Number of harmonics for ELL1H shapiro delay.",
             )
         )
+        self.remove_param("M2")
```

Comment:
Should we have something that can covert H3/H4 to SINI, M2?

Reply:
Ok, Let us do it in another PR

Reply:
I don't think that's a useful part of this PR, but I do think it should go into PINT. I can make those functions their own PR if that would be useful.